src/components/DoctorProfile/DoctorProfile.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+// import { Container } from 'react-bootstrap';
+import './DoctorProfile.scss';
+import DoctorProfilePic from './DoctorProfilePic.svg';
+
+function DoctorProfile() {
+  return (
+    <div>
+      {' '}
+      <div className="Doctor-profile-header">
```

Comment:
we are using the camelCase technique for naming. so change  Doctor-profile-header to DoctorProfileHeader.

**fix here and all**